src/CMakeLists.txt

```diff
@@ -1,13 +1,13 @@
 cmake_minimum_required(VERSION 3.1)
+project(cycloid)
+
```

Comment:
@a1k0n I thought that project had to be called at beginning of the file. However, the [docs](https://cmake.org/cmake/help/latest/command/project.html) do not say anything like that, so I was wrong. I'll remove the change.

Reply:
It's probably better at the top; that's fine.